src/Components/Coins/CoinsButton.js

```diff
@@ -20,9 +20,13 @@ export default function CoinsButton() {
 
   return (
     <div id="coinsButtonContainer">
-      <Button onClick={onCoinsButtonClick} id="coinsButton">
+      <GeneralButton
+        id="coinsButton"
+        className="uppercased bordered"
```

Comment:
this passing className operation is a mess, no-one can understand that intuitively, please think of something else.